Mendelugram/Mendelugram/View/Comment/CommentTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CommentTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class CommentTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: CommentTableViewCell.self)
+
+    struct Input {
+        let avatar: UIImage
+        let username: String
+        let comment: String
+    }
+
+    @IBOutlet weak var avatarImageView: UIImageView!
+    @IBOutlet weak var commentLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        avatarImageView.layer.cornerRadius = avatarImageView.frame.width / 2
+        avatarImageView.layer.masksToBounds = true
+    }
+
+    func configure(input: Input) {
```

Comment:
I think that `func configure(with input: Input)` would be more in align with standard Swift naming.